src/docs/torque.md

```diff
@@ -237,7 +257,8 @@ operator '.shared_function_info=' macro StoreJSFunctionSharedFunctionInfo(JSFunc
 As described above, the fields definied in Torque classes generate C++ code that removes the need for duplicate boilerplate accessor and heap visitor code, e.g. the JSFunction class definition above generates the following C++-accessible field offsets in js-objects.h:
 
 ```cpp
-#define JSFUNCTION_FIELDS(V) \
+#define TORQUE_GENERATED_JSFUNCTION_FIELDS(V) \
+V(kStartOfPointerFieldsOffset, 0) \
```

Comment:
Let's leave kStartOfPointerFieldsOffset because it is going away soon and suggests that weak and strong pointer sections always form a continuous tagged field section, which is not the case.
```suggestion
```